features/account/customer_account/viewing_orders_history/viewing_payment_status_for_an_order.feature

```diff
@@ -0,0 +1,29 @@
+@customer_account
+Feature: Viewing payment status on the order show page
+    In order to know whether an order has already been paid, or not
+    As a Customer
+    I want to see payment's status of my order
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is a user "lucy@teamlucifer.com" identified by "dantesdreams"
+        And the store has a product "Angel T-Shirt" priced at "$39.00"
+        And the store has a product "Angel Mug" priced at "$19.00"
+        But the store has "DHL" shipping method with "$8.60" fee
+        And the store allows paying with "Cash on Delivery"
+        And I am logged in as "lucy@teamlucifer.com"
+        And I placed an order "#00000666"
```

Comment:
For the sake of this feature and the below scenarios we just need this step, We do not care what's inside of this order or what was its shipping method.

Reply:
cc @Zales0123 ;>

Reply:
Agreed, however, I would postpone an introduction of a mega-order-setup step to a separate PR - it's for sure an improvement to an existing Behat suite 🚀